hashicorp/waypoint
# ui: Regenerate libs from protos

## Why the change?

The main motivation is #3116, but also it’s been a while since we regenerated the libs and there are plenty of changes to test against.

## What’s the plan?

- [x] Account for `opaqueany` in the build process and UI dependency graph
- [x] Rebuild the dependencies
- [x] Improve documentation around this process

## What versions of the tools did you use?

| Tool | Version |
| --- | --- |
| protoc | 3.17.3 |
| ts-protoc-gen | 0.15.0 |
| protoc-gen-grpc-web | 1.3.1 |

## What does it look like?

Hopefully no changes!

## How do I test it?

1. Check out the branch
2. Install dependencies
3. Trying booting the app locally as you normally would (against Mirage and a real server)
4. Try out the new guidance in the README
